src/js/shield_defs.js

```diff
@@ -2572,12 +2581,6 @@ export function loadShields() {
     ["ASD", "TWP"],
     ["ATH", "Trimble"],
     ["FAI", "Violet"],
-    ["HOL", "Berlin"],
-    ["HOL", "Clark"],
-    ["HOL", "Knox"],
-    ["HOL", "Monroe"], // No black border in reality, but a border is needed for contrast.
-    ["HOL", "Paint"],
-    ["HOL", "Salt_Creek"],
```

Comment:
These townships appear to be deleted instead of moved. Is this intentional? From #810:

> The following Holmes County townships no longer need to have a “TWP” on top, because the county’s shields no longer look identical to the townships’ shields when simplified. Their entries should be moved up to the array that currently lists Holmes County:

> The county engineer [plans to offer](https://thebargainhunter.com/news/news/signs-signs-everywhere-signs) townships the same sign minus the county outline. If and when that happens, we would migrate the townships’ entries to the array for green square shields. There would still be no need for a “TWP” banner in this style, because the township shields would be a different shape than the county shields.

While new signage is planned, I don't see a reason to remove support for existing signage.

Reply:
Whoops, that was a mistake, I added them back in.